python/test/t_CubaIntegration_std.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env python
```

Comment:
I suggest to use [black](https://github.com/psf/black) to automatically format the code.

Reply:
Hi @mbaudin47 ,

Thanks a lot for looking at this PR. Latest commits will certainly have addressed some of your concerns. Further reworking will follow for the remaining ones.